src/Elyon.Fastly.Api.DomainServices/InfoSessionFollowUpService.cs

```diff
@@ -103,14 +104,27 @@ await _emailSenderService.SendInfoSessionFollowUpEmailAsync(receiver, specDto.Me
 
         private async Task SendCompanyOnboardingEmailAsync(InfoSessionFollowUpSpecDto specDto, OrganizationDto organization)
         {
+            var receiversCount = specDto.Receivers.ToList().Count;
```

Comment:
use 
var receriversList = specDto.Receivers.ToList() 
and then use this list whenever needed